src/command_modules/azure-cli-advisor/HISTORY.rst

```diff
@@ -3,6 +3,10 @@
 Release History
 ===============
 
+1.0.0
+++++++
+* GA release.
```

Comment:
Here you will want to call out whatever breaking changes were made. For example
```
* BC: `advisor configuration set` renamed to `advisor configuration update`
```
and so on.